tests/shutdown/05-auto/suite.rc

```diff
@@ -12,4 +12,4 @@
         P1D = foo
 [runtime]
     [[foo]]
-        script = /bin/true
+        script = /usr/bin/true
```

Comment:
Fails on Ubuntu for me.

```
kinow@ranma:~$ /usr/bin/true
bash: /usr/bin/true: No such file or directory
kinow@ranma:~$ which true
/bin/true
```

Travis has this same test failing too.

Reply:
(I like the branch name BTW :smile:)

Reply:
Sounds like it'll have to be option (2) then.